IntelliTect.TestTools.Selenate/GoogleSearch/GoogleBrowser.cs

```diff
@@ -11,7 +11,7 @@ public GoogleBrowser(BrowserType browser) : base(browser)
         {
         }
 
-        public IWebElement FindElement(By by)
+        public new IWebElement FindElement(By by)
```

Comment:
This is not the same as overriding the method. Is this intentional?

Reply:
That example is being removed (which I totally missed doing earlier, so thank you!)